TAO/tests/IDLv4/anonymous_types/test.idl

```diff
@@ -0,0 +1,17 @@
+/*
+ * This is just a simple test making sure anonymous types work in IDL4 since
+ * there is already a test file at tests/IDL_Test/anonymous.idl.
+ */
+
+struct TestStruct {
+  short short_array[10];
+  sequence<long> long_sequence;
+  sequence<long, 3> bounded_long_sequence;
+};
+
+// Another struct, different lengths
+struct AnotherTestStruct {
+  short short_array[9];
```

Comment:
Would be good to also have the same array type appear twice, to make sure the generated code doesn't have multiple-definition errors.

Reply:
Oh yea, good point.